### ISSUE TITLE
Can't install on a new Windows PC.

### ISSUE THREAD

**Issue by alexbolduc1980** [sshaw/itunes_store_transporter_web#16]
Hello, I was able to fully installed the ISTW on a computer to test it. Now want to installe it on another one but installation failed

iTMSTransporter installed
Ruby 2.4 installed

trying the ruby install.rb command, i'm prompt with the Select your database: So i choose 3 (SQLite) and here what I got

Installing...
bundle install --path vendor/bundle --without-test development --binstubs
Installation failed

Thanks if you have a clue

**Comment by alexbolduc1980** [comment]
Yes I follow the link

Installing iTMSTransporter
Installing Ruby 2.4 With DevKit
and launching the command ruby install.rb

Does I have another thing to do to install the underlying database? On my first machine I follow this instruction and all work great.

Can the fact as the first try I was behind a proxy so the command line installation failed and as I retry it will always fail? Does it exist a script to uninstall what could be badly install and simply start fresh?

Thanks a lot!

**Comment by sshaw** [comment]
> Does I have another thing to do to install the underlying database? 

Yes, you must install the underlying database driver: https://sqlite.org/download.html

> Can the fact as the first try I was behind a proxy so the command line installation failed and as I retry it will always fail? 

If you're behind a proxy then that may be an issue but again, I need to see the installation output.

> Does it exist a script to uninstall what could be badly install and simply start fresh?

No. Remove the source directory to start fresh.

**Comment by alexbolduc1980** [comment]
Thanks for the reply, trying to restart fresh without succes. So I try to simply copied the folder on the one machine where it work to the other and all good work like a charm!

Thanks a lot for the support!
